spec/middleware/rack_attack.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Rack::Attack do
+  include Rack::Test::Methods
+
+  def app
+    Rails.application
+  end
+
+  describe 'feedback submission limits' do
+    it 'responds with 429 after 5 requests' do
```

Comment:
Open to suggestions for breaking this functionality out to aid addition of future rate limiting specs.

Reply:
I can see value in some kind of helper class that eases this testing, but this doesn't seem burdensome for now. Would be nice in combination with a config-driven definition of the rate limits rather than having to keep that and specs in sync.